backend/backend/core/chains.py

```diff
@@ -32,39 +31,43 @@ class AdjacentQueriesChain(object):
 
     class SearchQueryList(BaseModel):
         """List of search queries adjacent to a common query."""
+
         queries: List[str]
 
-    def __init__(self, model):
+    def __init__(self, models):
```

Comment:
The new code has removed the `model` parameter from the `__init__` method, which could lead to bugs if not properly handled.